LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentApi.kt

```diff
@@ -0,0 +1,84 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.core.BraintreeClient
+import com.braintreepayments.api.localpayment.LocalPaymentNonce.Companion.fromJSON
+import org.json.JSONException
+import org.json.JSONObject
+
+internal class LocalPaymentApi(private val braintreeClient: BraintreeClient) {
+    fun createPaymentMethod(
+        request: LocalPaymentRequest,
+        callback: LocalPaymentInternalAuthRequestCallback
+    ) {
+        val returnUrl =
+            braintreeClient.getReturnUrlScheme() + "://" + LocalPaymentClient.LOCAL_PAYMENT_SUCCESS
+        val cancel =
+            braintreeClient.getReturnUrlScheme() + "://" + LocalPaymentClient.LOCAL_PAYMENT_CANCEL
+
+        val url = "/v1/local_payments/create"
+        braintreeClient.sendPOST(url = url, data = request.build(returnUrl, cancel),
+            responseCallback = { responseBody: String?, httpError: Exception? ->
```

Comment:
We should be able to move the lambda callback outside of the `sendPost()` parenthesis since the callback parameter is the last in the list: https://kotlinlang.org/docs/lambdas.html#passing-trailing-lambdas